### PR TITLE
feat: blog update

### PR DESCRIPTION
**Pull request by rachelhox**

### CODE REVIEW

File: src/hooks/get_hk_timezone.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+export const useHKT = (publishedAt: string) => {
+  const [time, setTime] = useState(publishedAt);
+  useEffect(() => {
+    const hkt = moment
+      .tz(publishedAt, 'Do MMM YYYY, h:mm a', 'Asia/Hong_Kong')
+      .format('Do MMM YYYY, h:mm a');
+    setTime(hkt);
+  }, [publishedAt]);
+  return time;
+};
```
Comment (teamchong) [inline src/hooks/get_hk_timezone.ts:13]: I noticed a small deviation from our code standard in this PR. As per our guidelines, the `useHKT` function should be set as the default export, and the filename should be `useHKT.ts`.

**Review by teamchong — APPROVED** [review]
LGTM

File: src/hooks/get_hk_timezone.ts (hunk 1 above)
Comment (rachelhox) [reply src/hooks/get_hk_timezone.ts]: changed the file name for `useHKT` hook, and exported the hook as default export. Will work on refactoring the import for all custom hooks on the code base, so for the code base consistency will import this hook from `@hooks` for now. Please let me know if there's any issues! @teamchong

**Review by teamchong — COMMENTED** [review]
LGTM

**Pull request merged**